Cargo.toml

```diff
@@ -50,6 +50,8 @@ petgraph = { version = "0.6.3", default-features = false }
 context-iterators = "0.2.0"
 serde_json = "1.0.97"
 delegate = "0.10.0"
+rustversion = "1.0.14"
+paste = "1.0"
```

Comment:
I continue my brazen injecting of dtolnay crates

Reply:
Note this one is already in `dev-dependencies`